core/block.js

```diff
@@ -1357,3 +1357,26 @@ Blockly.Block.prototype.moveBy = function(dx, dy) {
 Blockly.Block.prototype.makeConnection_ = function(type) {
   return new Blockly.Connection(this, type);
 };
+
+/**
+ * Checks whether all of the required inputs are filled.
```

Comment:
Looks good to me, except for this comment.  "required inputs" implies that some inputs are required and others not.  Suggest changing this to:
 \* Checks whether all inputs are filled.
Also, the return line should be:
 \* @return {boolean} True if all inputs are filled.
Same applies to the relevant comments in Workspace.

I'm not going to merge this PR (since I'm on paternity leave), but I expect the team at Google will.